airbrake/handler.py

```diff
@@ -25,7 +25,7 @@ class AirbrakeHandler(logging.Handler):
     # pylint: disable=too-many-arguments
 
     def __init__(self, airbrake=None, level=logging.ERROR, project_id=None,
-                 api_key=None, environment=None, base_url=None, hostname=None):
+                 api_key=None, **config):
```

Comment:
Oh ya for sure, hit me with it 😄 

We could also do something like a settings file, or pass in some sort of config dict. I see what you're saying though, there's a difference between client configuration and things that add context to the payload. Separating the two is a bit more work, so I think I'm leaning towards this for the short term so I can focus on features.